utils/forest_shim/src/address.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_shared::address::Address as Address_v2;
+use fvm_shared3::address::Address as Address_v3;
+use std::ops::{Deref, DerefMut};
+
+pub struct Address(Address_v3);
```

Comment:
Should we add the derives from the child? E.g. `#[derive(Copy, Clone, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)`.

Reply:
We should.